hooks/test/test.go

```diff
@@ -39,12 +39,24 @@ func NewLocal(logger *logrus.Logger) *Hook {
 
 }
 
+type TestOption func(logger *logrus.Logger)
+
+func FatalPanics(logger *logrus.Logger) {
+	logger.Exit = func(code int) {
+		panic(code)
+	}
+}
+
 // NewNullLogger creates a discarding logger and installs the test hook.
-func NewNullLogger() (*logrus.Logger, *Hook) {
+func NewNullLogger(options ...TestOption) (*logrus.Logger, *Hook) {
```

Comment:
I'd rather not change this interface by adding a new parameter. We can just patch the resulting logger when needed by calling the same function on the returned object